modules/globebrowsing/tile/chunktile.h

```diff
@@ -34,6 +34,12 @@ namespace openspace {
 namespace globebrowsing {
 
 struct ChunkTile {
+	ChunkTile() : tile(Tile::TileUnavailable) {};
```

Comment:
If all of the substructs have default constructors (or default values), that should be possible.